### DISCUSSION TITLE
Wishlist for next PCB revision

### DISCUSSION THREAD

**Discussion by ericlathrop** [ElectroTechnique/electrotechnique.github.io#76] [New developments]
Having built the kit, here's a few things I'd like to see in the next revision:

1. Separate 9v center negative barrel jack for power (like a guitar pedal). To avoid USB ground loops.
2. Dual 1/4" TS jacks for separate left/right audio outputs.
3. SD card access. Having to disassemble the case to get at the card is annoying.
4. Attaching the USB to the Teensy via a normal USB cable that's soldered onto the TSynth PCB. The little pads on the underside of the Teensy are a PITA to solder to, and it makes it difficult to swap out the Teensy.
5. Building the audio shield into the Teensy PCB. Hooking up the audio shield is annoying, and also prevents swapping out the Teensy once the audio jack is attached.

**Comment by Electrotechnique** [comment]
Hello, I designed a version based on Teensy MicroMod, which features a PCM5102A DAC on a PCB (Aliexpress etc) and SD card near the edge. This eliminates much of the unpleasant soldering of the audio board and USB pads. A 9V socket can be added instead of the optional on/off switch. L/R 1/4" jacks would involve moving things about. It can mostly be assembled by JLCPCB/PCBWay, although the M.2 socket comes and goes from stock, there are plenty of SD card socket choices.

I haven't actually got this made yet. Is it time to open source it? The current software does work with it with changes made for the new DAC and different pins.
![aaa](https://user-images.githubusercontent.com/50934180/137606182-400a94f0-9539-4d94-a959-32d6ccd96d14.jpg)

**Reply by ericlathrop** [reply]
Nice! That looks like a solid improvement! I just get annoyed having to hook adapters up to everything. Having a stereo line out means I'll need a stereo breakout adapter because all the audio interfaces I've seen only have mono inputs.

**Comment by rolfdegen** [comment]
Hallo ET

The new PCB is a great idea. I also use a PCM5102A PCB in my Jeannie. It is inexpensive and has an output voltage of over 5Vpp. 
The built-in voltage regulator on the DAC PCB means there is hardly any interference. For the output amplifier I use + -8V with a charge pump. It works very well. Power supply is 12VDC

You can see the circuit diagrams here: https://www.tubeohm.com/jeannie.html

Greetings from Germany. Rolf Degen
